.rustfmt.toml

```diff
@@ -0,0 +1,6 @@
+indent_style = "Visual"
```

Comment:
I like `"Block"` much better…

Reply:
I do too. I think that the visual indentation makes the code harder to read more often than it makes it easier.